core/src/test/java/com/radixdlt/api/core/NetworkConfigurationTest.java

```diff
@@ -67,32 +67,39 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import com.radixdlt.api.DeterministicCoreApiTestBase;
+import com.radixdlt.api.core.generated.models.AccessRulesModuleFieldOwnerRoleSubstate;
+import com.radixdlt.api.core.generated.models.DenyAllAccessRule;
 import com.radixdlt.api.core.generated.models.StatePackageRequest;
 import org.junit.Test;
 
 public class NetworkConfigurationTest extends DeterministicCoreApiTestBase {
   @Test
-  @SuppressWarnings("try")
   public void test_network_configuration() throws Exception {
     try (var test = buildRunningServerTest()) {
       test.suppressUnusedWarning();
 
-      final var response = getStatusApi().statusNetworkConfigurationPost();
+      final var configurationResponse = getStatusApi().statusNetworkConfigurationPost();
 
-      assertThat(response.getNetwork())
+      assertThat(configurationResponse.getNetwork())
           .isEqualTo(DeterministicCoreApiTestBase.networkDefinition.logical_name());
-      assertThat(response.getNetworkHrpSuffix())
+      assertThat(configurationResponse.getNetworkHrpSuffix())
           .isEqualTo(DeterministicCoreApiTestBase.networkDefinition.hrp_suffix());
 
       // And check the package endpoint whilst we're here, using a well known address...
-      final var faucetPackageAddress = response.getWellKnownAddresses().getFaucetPackage();
+      final var faucetPackageAddress =
+          configurationResponse.getWellKnownAddresses().getFaucetPackage();
 
-      // TODO(wip): it's not so easy to assert on schema anymore; this only tests "no exception":
```

Comment:
_(note from author)_

This was my fault (a `TODO(wip)` comes from me and my internal rule says that such expression must not be pushed - maybe I should make it a git hook or something).
I now addressed it in the only way I saw viable (still lame). Suggestions (on "what to assert better here") welcome.